### PR TITLE
created category box

### PR DESCRIPTION
**Pull request by Poojyabadramraju**

### CODE REVIEW

File: src/components/CategoryBox.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <div className="flex flex-wrap justify-around items-center content-center gap-8 my-5">
+        {CategoryItems.map((item,index)=>{
+            return(
+                <div key={index} className="flex flex-col justify-center items-center gap-2 min-w-[100px] w-[250px] max-lg:w-[200px] max-md:w-[150px] max-sm:w-[100px]">
+                    <img src={item.img} alt={item.name} 
+                        className="self-start rounded-tl-[20px] rounded-tr-[90px] rounded-br-[20px] rounded-bl-[90px] max-md:rounded-bl-[60px] max-md:rounded-tr-[60px] max-sm:rounded-bl-[40px] max-sm:rounded-tr-[40px] w-full h-full aspect-square "
+                    />
+                    <p className="font-medium text-xl max-lg:text-base max-md:text-sm text-center">{item.name}</p>
+                </div>
```
Comment (ketanbaitule) [inline src/components/CategoryBox.jsx:11]: Convert this into different component

Comment (ketanbaitule) [reply src/components/CategoryBox.jsx:11]: CategoryIcon.jsx in src/component/

File: src/components/CategoryBox.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    </div>
+  )
+};
+
+export default CategoryBox;
+
+const CategoryItems=[
+    {
+        name:"Soups",
+        img:"soups.jpg",
```
Comment (ketanbaitule) [inline src/components/CategoryBox.jsx:23]:
```suggestion
        img:"/soups.jpg",
```

Pls do same for all

File: src/components/CategoryBox.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                </div>
+            )
+        })}
+    </div>
+  )
+};
+
+export default CategoryBox;
+
+const CategoryItems=[
```
Comment (ketanbaitule) [inline src/components/CategoryBox.jsx:20]:
```suggestion
const categoryItems=[
```

First Letter Capital generally denotes an Element

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/components/Categories/CategoryBox.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import CategoryIcon from "./CategoryIcon";
+
+const CategoryBox = () => {
+  return (
+    <div className="flex flex-wrap justify-around items-center content-center gap-8 my-5">
+      {categoryItems.map((category) => {
+        return (
+          <CategoryIcon
```
Comment (ketanbaitule) [inline src/components/Categories/CategoryBox.jsx:8]:
```suggestion
      {categoryItems.map((category, index) => {
        return (
          <CategoryIcon
           key={index}
```

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/components/Categories/CategoryBox.jsx (hunk 4 above)
Comment (Poojyabadramraju) [reply src/components/Categories/CategoryBox.jsx]: Done sir

**Review by ketanbaitule — APPROVED** [review]

**Pull request merged**